docs/samples/Microsoft.ML.Samples/Dynamic/ImageAnalytics/ExtractPixels.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.IO;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ExtractPixelsExample
+    {
+        public static void ExtractPixels()
```

Comment:
I added it; it will show up on the next iteration. Personally, i don' think that is very important, because those samples will display in the website that doccuments the components, so in context, hopefully there is enough explanation. 

---
In reply to: [253609946](https://github.com/dotnet/machinelearning/pull/2372#discussion_r253609946) [](ancestors = 253609946)